patches/common/chromium/spellcheck_worditerator.patch

```diff
@@ -0,0 +1,19 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Fri, 26 Oct 2018 08:29:57 +0200
+Subject: spellcheck_worditerator.patch
+
+Fix build error
+
+diff --git a/components/spellcheck/renderer/spellcheck_worditerator.cc b/components/spellcheck/renderer/spellcheck_worditerator.cc
+index 1bd631cbd85a88c14883844c73999c675233321c..7ab4be4af748e5f584719105ac042daeb5cf3896 100644
+--- a/components/spellcheck/renderer/spellcheck_worditerator.cc
++++ b/components/spellcheck/renderer/spellcheck_worditerator.cc
+@@ -16,7 +16,6 @@
+ #include "base/macros.h"
+ #include "base/strings/stringprintf.h"
+ #include "base/strings/utf_string_conversions.h"
+-#include "components/spellcheck/renderer/spellcheck.h"
```

Comment:
In that case why not add `//components/spellcheck/renderer` as a dep for `electron_lib` ? I don't think this is a necessary patch.

Reply:
My original assumption was `spellcheck_worditerator` was a standalone source, I would prefer building the component as such.

Reply:
this includes some mojo stuff, which fails. This include was actually removed in our copy.

Reply:
@deepak1556
```
In file included from ../../components/spellcheck/renderer/spellcheck_worditerator.cc:19:
../../components/spellcheck/renderer/spellcheck.h:19:10: fatal error: 'components/spellcheck/common/spellcheck.mojom.h' file not found
#include "components/spellcheck/common/spellcheck.mojom.h"
```

Reply:
> This include was actually removed in our copy.

Agreed, but the source copies were trying to be minimal. I think we can build the component now.

Reply:
Yeah deps should be

```
//components/spellcheck/common",
"//components/spellcheck/renderer",
```

since the renderer has an explicit dep on common.

Reply:
Lets hear thoughts from other maintainers on this /cc @nornagon , @nitsakh since you have been rewriting the spellcheck implementation.

Reply:
@deepak1556 ok, I will try to add it as a component and see if it works

Reply:
@deepak1556 The component works, updated the PR. Let's wait for others as suggested.